wolfcrypt/src/asn.c

```diff
@@ -35469,6 +35469,21 @@ static int GetRevoked(const byte* buff, word32* idx, DecodedCRL* dcrl,
 
     end = *idx + len;
 
+#ifdef CRL_STATIC_REVOKED_LIST
+    int totalCerts = dcrl->totalCerts;
```

Comment:
If I am reading this right, you'll have the `totalCerts` variable declaration in middle of code. Needs to be at top of function or brace.